feathr-example/bureau_bureau-balance.ipynb

```diff
@@ -0,0 +1,1224 @@
+      "2022-09-02 11:20:31.207 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: running\n",
+      "2022-09-02 11:21:01.484 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: running\n",
+      "2022-09-02 11:21:31.785 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: running\n",
+      "2022-09-02 11:22:02.158 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: running\n",
+      "2022-09-02 11:22:32.429 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: running\n",
+      "2022-09-02 11:23:02.749 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: success\n"
+     ]
+    }
+   ],
+   "source": [
+    "feature_queries = [\n",
+    "    FeatureQuery(\n",
+    "        feature_list=[\n",
+    "            \"f_SK_ID_CURR\",\n",
+    "            \"f_SK_ID_BUREAU\",\n",
+    "            \"f_CREDIT_ACTIVE\",\n",
+    "            \"f_CREDIT_CURRENCY\",\n",
+    "            \"f_DAYS_CREDIT\",\n",
+    "            \"f_CREDIT_DAY_OVERDUE\",\n",
+    "            \"f_DAYS_CREDIT_ENDDATE\",\n",
+    "            \"f_DAYS_ENDDATE_FACT\",\n",
+    "            \"f_AMT_CREDIT_MAX_OVERDUE\",\n",
+    "            \"f_CNT_CREDIT_PROLONG\",\n",
+    "            \"f_AMT_CREDIT_SUM\",\n",
+    "            \"f_AMT_CREDIT_SUM_DEBT\",\n",
+    "            \"f_AMT_CREDIT_SUM_LIMIT\",\n",
+    "            \"f_AMT_CREDIT_SUM_OVERDUE\",\n",
+    "            \"f_CREDIT_TYPE\",\n",
+    "            \"f_DAYS_CREDIT_UPDATE\",\n",
+    "            \"f_AMT_ANNUITY\",\n",
+    "\n",
+    "            \"f_NUM_CREDIT_COUNT\",\n",
+    "            \"f_DEBT_CREDIT_RATIO\",\n",
+    "        ], key=key_SK_ID_BUREAU),\n",
+    "    \n",
+    "    FeatureQuery(\n",
+    "        feature_list=[\n",
+    "            \"f_MONTHS_BALANCE\",\n",
+    "            \"f_STATUS\",\n",
+    "            \"f_CREDIT_STATUS_EMA_AVG\"\n",
+    "        ], key=key_SK_ID_BUREAU),\n",
+    "    \n",
+    "    FeatureQuery(\n",
+    "        feature_list=[\n",
+    "            \"f_CREDIT_STATUS_EMA_AVG_DER\"\n",
+    "        ], key=key_SK_ID_BUREAU),\n",
+    "]\n",
+    "\n",
+    "# spine dataset was created manually, it's the same as the bureau.csv \n",
+    "# but with an added column named TRAN_DATE with a value of \n",
+    "# datetime.datetime(2021,1,1,11,34,44).strftime('%Y-%m-%d %X') (see. ./scripts/add_time_to_csv.py)\n",
+    "settings = ObservationSettings(\n",
+    "    observation_path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/bureau_all_time.csv\",\n",
```

Comment:
Azure storage account reference

Reply:
Moved the resource prefix to a variable.